execution/src/windows/execution.cc

```diff
@@ -106,6 +107,10 @@ namespace leatherman { namespace execution {
     static tuple<scoped_handle, scoped_handle> CreatePipeThrow()
     {
         static LONG counter = 0;
+        static auto rd_gen = []() {
+            std::random_device rd;
```

Comment:
Uses the system's best random number generator to generate a seed for a pseudo-random number generator. random_device is slow, so we only use it to generate the seed.